[v1.1.2] Longhorn UI use `Extension developer features` (#277)
harvester/docs: versioned_docs/version-v1.1/troubleshooting/harvester.md

_Available as of v1.1.0_

> You can now access the embedded Rancher and Longhorn dashboards directly on the `Support` page, but you must enable the `Extension developer features` first via the Preference page. (In Harvester v1.1.0 and v1.1.1, the option name is `Developer Tools & Features`) 

Reviewer: ```suggestion
You can now access the embedded Rancher and Longhorn dashboards directly on the `Support` page, but you must first go to the `Preferences` page and select `Enable Extension developer features` under `Advanced Features` (in Harvester v1.1.0 and v1.1.1, the option name is `Developer Tools & Features`).
```
